### PR TITLE
Use same pom with different profile to split 4.x and 6.x version

### PR DESCRIPTION
**Pull request by fangjian0423**
## Purpose

Resolve https://github.com/Azure-Samples/azure-spring-boot-samples/issues/623.

1. As title said, use sample pom with different profile to split 4.x and 6.x version
2. Update README, add `Run with Spring Boot 3` section

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

**Comment by fangjian0423** [comment]
@chenrujun @moarychan hi all, please take a look.

File: aad/spring-security/servlet/oauth2/login-authenticate-using-private-key-jwt/src/test/java/com/azure/spring/sample/reactive/servlet/oauth2/login/jwt/azure/activedirectory/AzureActiveDirectoryCertificateSignedAssertionFactoryTest.java [hunk 1]
```diff
@@ -24,12 +24,12 @@ public void testPfx() throws AzureActiveDirectoryAssertionException {
...
     // TODO support pem file.
-    @Disabled("Pem file is not supported now.")
-    @Test
+     @Disabled("Pem file is not supported now.")
+     @Test
     public void testPem() throws AzureActiveDirectoryAssertionException {
-        test(new AzureActiveDirectoryCertificateSignedJwtAssertionFactory(
-            "src/test/resources/encrypted-private-key-and-certificate.pem", "myPassword1", TEST_TENANT_ID,
-            TEST_CLIENT_ID));
+//        test(new AzureActiveDirectoryCertificateSignedJwtAssertionFactory(
```
Comment (fangjian0423) [inline aad/spring-security/servlet/oauth2/login-authenticate-using-private-key-jwt/src/test/java/com/azure/spring/sample/reactive/servlet/oauth2/login/jwt/azure/activedirectory/AzureActiveDirectoryCertificateSignedAssertionFactoryTest.java:30]: comment the `@Disabled` and `@Test` annotation doesn't work, check it [here](https://github.com/fangjian0423/azure-spring-boot-samples/actions/runs/3763007207/jobs/6396183014) . So I comment the code.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -21,8 +21,8 @@ jobs:
         distribution: 'adopt'
         cache: maven
     - name: Build azure-spring-boot-samples-4.x with Maven - Java ${{ matrix.java-version }}
-      run: mvn -f azure-spring-boot-samples-4.x-pom.xml --batch-mode --update-snapshots verify
+      run: mvn -f pom.xml -P spring-cloud-azure-4.x --batch-mode --update-snapshots verify
```
Comment (chenrujun) [inline .github/workflows/ci.yml:24]: Is it possible to delete `-P spring-cloud-azure-4.x` by using `active by default`?
Refs: https://maven.apache.org/guides/introduction/introduction-to-profiles.html

File: CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3_TEMPLATE.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 
 - [JDK17](https://www.oracle.com/java/technologies/downloads/).
 
-## 1.2 Update the pom parent
-   
-   Note: The pom parent of the samples should point to *azure-spring-boot-samples-6.x-pom.xml*.
+## 1.2 Update dependencies(optional)
 
-## 1.3 Update dependencies(optional)
+This is written in each sample's README. Most of them don't need to update.
```
Comment (chenrujun) [inline CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3_TEMPLATE.md:9]: How about delete this line? This file `CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3.md` is used to tell customer how to convert sample to use Spring Boot 3. So, these steps should be written in `CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3.md` instead of README.md.

File: CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3_TEMPLATE.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
 
-## 1.4 Update Java code(optional)
+## 1.3 Update Java code(optional)
+
+This is written in each sample's README. Most of them don't need to update.
```
Comment (chenrujun) [inline CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3_TEMPLATE.md:13]: Same here.

File: README.md [hunk 5]
```diff
@@ -14,11 +14,36 @@ Two Maven profiles have been defined in this project to support compiling Spring
...
+### Convert samples to use Spring Boot 3
+
 To get samples for Spring Boot 3, please refer to the content in each sample's *README.md* like this:
-> Current sample is using Spring Cloud Azure 4.x (which is compatible with Spring Boot 2.x). 
-> If you want sample about Spring Cloud Azure 6.x (which is compatible with Spring Boot 3.x), 
+> Current sample is using Spring Cloud Azure 4.x (which is compatible with Spring Boot 2.x).
+> If you want sample about Spring Cloud Azure 6.x (which is compatible with Spring Boot 3.x),
 > please refer to [CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3.md](./CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3_TEMPLATE.md).
 
+If there is no such instruction, the example can directly activate the profile of Spring Cloud Azure 6.x to run.
```
Comment (chenrujun) [inline README.md:24]: How about telling customers explicitly like this:

> Current sample can work with both Spring Cloud Azure 4.x (which is compatible with Spring Boot 2.x) and Spring Cloud Azure 6.x (which is compatible with Spring Boot 3.x). ...

File: README.md [hunk 6]
```diff
@@ -14,11 +14,36 @@ Two Maven profiles have been defined in this project to support compiling Spring
...
+
 If you're adding a new example, here's a template that converts the example to support Spring Boot 3: [./CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3_TEMPLATE.md](CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3_TEMPLATE.md).
 
-**Note**: This section is used to convert the samples to run in Spring Boot 3, the branch `feature/spring-boot-3` is no longer maintained.
\ No newline at end of file
+**Note**: This section is used to convert the samples to run in Spring Boot 3, the branch `feature/spring-boot-3` is no longer maintained.
```
Comment (chenrujun) [inline README.md:28]: How about moving this line to the line below `## Samples for Spring Boot 3`

File: aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server/CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+    Current for Spring Boot 2:
+
+    ```java
+    import com.azure.spring.cloud.autoconfigure.aad.AadJwtBearerTokenAuthenticationConverter;
+    import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
+    import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+    import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+    import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+    
+    @EnableWebSecurity
```
Comment (chenrujun) [inline aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server/CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3.md:20]: How about moving this into `src/main/java` and commend out some codes out like `@EnableWebSecurity` and `@EnableGlobalMethodSecurity`  to make these codes not take effect.

And in `CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3.md`, let customer remove `//` manually.

Comment (chenrujun) [reply aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server/CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3.md:20]: BTW, will CI pass without these changes in Spring Boot 3?

File: aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter-stateless/README.md [hunk 8]
```diff
@@ -115,6 +115,11 @@ mvn spring-boot:run
 #### Demo
 ![demoonstration video](docs/demo.webp "Demo Video")
```
Comment (chenrujun) [inline aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter-stateless/README.md:117]: @saragluna Do we still keep this feature in Spring Cloud Azure 6?

File: aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+    ```java
+    import javax.servlet.http.HttpServletRequest;
+    import javax.servlet.http.HttpServletResponse;
+    ```
+
+  Updated for Spring Boot 3 to the following:
+
+    ```java
+    import jakarta.servlet.http.HttpServletRequest;
+    import jakarta.servlet.http.HttpServletResponse;
```
Comment (chenrujun) [inline aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3.md:22]: Same here. Will CI fail without these changes?

File: aad/spring-security/servlet/oauth2/login-authenticate-using-private-key-jwt/src/test/java/com/azure/spring/sample/reactive/servlet/oauth2/login/jwt/azure/activedirectory/AzureActiveDirectoryCertificateSignedAssertionFactoryTest.java (hunk 1 above)
Comment (chenrujun) [reply aad/spring-security/servlet/oauth2/login-authenticate-using-private-key-jwt/src/test/java/com/azure/spring/sample/reactive/servlet/oauth2/login/jwt/azure/activedirectory/AzureActiveDirectoryCertificateSignedAssertionFactoryTest.java:30]: It's weird, could you please do some more investigation?

File: pom.xml [hunk 10]
```diff
@@ -0,0 +1,506 @@
...
+            <type>pom</type>
+            <scope>import</scope>
+          </dependency>
+          <dependency>
+            <groupId>com.azure.spring</groupId>
+            <artifactId>spring-cloud-azure-dependencies</artifactId>
+            <version>${version.spring.cloud.azure}</version>
+            <type>pom</type>
+            <scope>import</scope>
+          </dependency>
```
Comment (chenrujun) [inline pom.xml:219]: Is it possible to move these contents out of `profile`? Because these contents exist for both spring-cloud-azure-4 and spring-cloud-azure-6.

File: pom.xml [hunk 11]
```diff
@@ -0,0 +1,506 @@
...
+          <name>Spring Milestones</name>
+          <url>https://repo.spring.io/milestone</url>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+        </pluginRepository>
+      </pluginRepositories>
+    </profile>
+    <profile>
+      <id>buildpack-4x</id>
```
Comment (chenrujun) [inline pom.xml:258]: For profiles except `spring-cloud-azure-4` and `spring-cloud-azure-6`, the `modules` are empty, is it right?

Comment (chenrujun) [reply pom.xml:258]: Is it valid to active `buildpack-4x` and `spring-cloud-azure-6` at the same time? It should be written down in README.md, otherwise our customers will be confused.

File: aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server/CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3.md (hunk 7 above)
Comment (fangjian0423) [reply aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server/CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3.md:20]: After discussion, won't update this.

File: aad/spring-security/servlet/oauth2/login-authenticate-using-private-key-jwt/src/test/java/com/azure/spring/sample/reactive/servlet/oauth2/login/jwt/azure/activedirectory/AzureActiveDirectoryCertificateSignedAssertionFactoryTest.java (hunk 1 above)
Comment (fangjian0423) [reply aad/spring-security/servlet/oauth2/login-authenticate-using-private-key-jwt/src/test/java/com/azure/spring/sample/reactive/servlet/oauth2/login/jwt/azure/activedirectory/AzureActiveDirectoryCertificateSignedAssertionFactoryTest.java:30]: > It's weird, could you please do some more investigation?

check it [here](https://stackoverflow.com/a/55864867/4239209), seems caused by incompatibilities between `maven-surefire-plugin` and `junit5`.

upgrade `maven-surefire-plugin` to 2.22.0 resolve this problem.

File: README.md [hunk 12]
```diff
@@ -4,21 +4,131 @@
...
 
+## All Samples in This Repo
+
+### Azure Active Directory
+
+| Sample Project                                                                                                                                                                                                                                                 | Support Spring Cloud Azure 4.x     | Support Spring Cloud Azure 6.x | 
+|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------|--------------------------------|
+| [aad-b2c-resource-server](aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server)                                                                                                                                                         | Yes                                | No                             |
```
Comment (chenrujun) [inline README.md:13]: How about using emoji instead of `Yes` and `No`?

:white_check_mark:
:x:

Refs: https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md

File: README.md [hunk 13]
```diff
@@ -4,21 +4,131 @@
...
 
 ## Run Samples Based On Spring Native in Spring Boot 2
-Two Maven profiles have been defined in this project to support compiling Spring applications to native executables: `buildpack` and `native`. The `buildpack` profile will use [Buildpacks](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#getting-started-buildpacks) and the `native` profile will use [Native Build Tools](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#getting-started-native-build-tools). Please follow the [storage-blob-native sample](spring-native/storage-blob-native) for more details.
+Two Maven profiles have been defined in this project to support compiling Spring applications to native executables: `buildpack-4.x` and `native-4.x`. The `buildpack-4.x` profile will use [Buildpacks](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#getting-started-buildpacks) and the `native-4.x` profile will use [Native Build Tools](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#getting-started-native-build-tools). Please follow the [storage-blob-native sample](spring-native/storage-blob-native) for more details.
```
Comment (chenrujun) [inline README.md:96]: `buildpack-4.x` and `native-4.x` are still confusing. By adding `-4.x`, Do you mean it only work for Spring Cloud Azure 4.x? Customer will not be sure about what `-4.x` mean.

File: pom.xml [hunk 14]
```diff
@@ -0,0 +1,499 @@
...
+      </dependencyManagement>
+      <repositories>
+        <repository>
+          <id>spring-release</id>
+          <name>Spring release</name>
+          <url>https://repo.spring.io/release</url>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+        </repository>
```
Comment (chenrujun) [inline pom.xml:169]: Why is this necessary?

File: pom.xml [hunk 15]
```diff
@@ -0,0 +1,499 @@
...
+      </repositories>
+      <pluginRepositories>
+        <pluginRepository>
+          <id>spring-release</id>
+          <name>Spring release</name>
+          <url>https://repo.spring.io/release</url>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+        </pluginRepository>
```
Comment (chenrujun) [inline pom.xml:179]: Why is this necessary?

File: pom.xml [hunk 16]
```diff
@@ -0,0 +1,499 @@
...
+          <name>Spring Milestones</name>
+          <url>https://repo.spring.io/milestone</url>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+        </pluginRepository>
+      </pluginRepositories>
+    </profile>
+    <profile>
+      <id>buildpack-4.x</id>
```
Comment (chenrujun) [inline pom.xml:251]: If `-4.x` means `spring-cloud-azure-4.x`, how about renaming to `build-pack-for-spring-cloud-azure-4.x`?

File: pom.xml [hunk 17]
```diff
@@ -0,0 +1,499 @@
...
+              <buildArgs>
+                <buildArg>-H:+ReportExceptionStackTraces</buildArg>
+              </buildArgs>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <profile>
+      <id>native-6.x</id>
```
Comment (chenrujun) [inline pom.xml:404]: Same here.

File: pom.xml [hunk 18]
```diff
@@ -0,0 +1,499 @@
...
+      <build>
+        <pluginManagement>
```
Comment (chenrujun) [inline pom.xml:406]: Why it's `pluginManagement`, not `plugin`?

File: pom.xml [hunk 19]
```diff
@@ -0,0 +1,499 @@
...
+                  </goals>
+                </execution>
+              </executions>
+            </plugin>
+          </plugins>
+        </pluginManagement>
+      </build>
+    </profile>
+    <profile>
+      <id>nativeTest-6.x</id>
```
Comment (chenrujun) [inline pom.xml:453]: 1. Why nativeTest is only necessary for `6.x`?
2. Should we active this profile in GitHub actions?
3. Could you please add more description about spring-native related profiles? These profiles make the pom file too long, may make customer confused.

File: spring-native/storage-blob-native/README.md [hunk 20]
```diff
@@ -154,10 +154,10 @@ NOTE: If you want to build a lightweight container containing a native executabl
 
 Add the `spring.cloud.azure.storage.blob.account-key` configuration, and replace the relevant values in *application.yml* according to the saved output variable value. You can find these values in the temp file *terraform/terraform.tfstate*, or you can visit the Azure portal to get them.
 
-Run `mvn -Pbuildpack package spring-boot:build-image`, see [Build the native application](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#_build_the_native_application) for more details.
+Run `mvn -P spring-cloud-azure-4.x,buildpack-4.x package spring-boot:build-image`, see [Build the native application](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#_build_the_native_application) for more details.
```
Comment (chenrujun) [inline spring-native/storage-blob-native/README.md:157]: It's better to add these contents in `spring-native/README.md`.

File: spring-native/storage-blob-native/README.md [hunk 21]
```diff
@@ -175,10 +175,10 @@ GraalVM `native-image` compiler should be installed, see [System Requirements](h
 
 - Build the native application
 
-Run `mvn -Pnative -DskipTests package` command using `x64 Native Tools Command Prompt`, see [Build the native application](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#_build_the_native_application_2) for more details.
+Run `mvn -P spring-cloud-azure-4.x,native-4.x -DskipTests package` command using `x64 Native Tools Command Prompt`, see [Build the native application](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#_build_the_native_application_2) for more details.
 
 ```shell
-mvn -Pnative -DskipTests package
+mvn -P spring-cloud-azure-4.x,native-4.x -DskipTests package
```
Comment (chenrujun) [inline spring-native/storage-blob-native/README.md:181]: All these descriptions are about native-4.x, we lack content about native-6.x.

File: spring-petclinic-microservices/pom.xml [hunk 22]
```diff
@@ -38,6 +38,8 @@
     <webjars-jquery.version>3.1.1-1</webjars-jquery.version>
     <webjars-angular.version>1.6.4</webjars-angular.version>
     <webjars-angular-ui-router.version>1.0.30</webjars-angular-ui-router.version>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
```
Comment (chenrujun) [inline spring-petclinic-microservices/pom.xml:42]: Why is this necessary?

File: spring-petclinic-microservices/pom.xml [hunk 23]
```diff
@@ -111,6 +113,7 @@
           <plugin>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-maven-plugin</artifactId>
+            <version>${version.spring.boot}</version>
```
Comment (chenrujun) [inline spring-petclinic-microservices/pom.xml:116]: Same here. Why didn't it cause failure before this PR?

File: pom.xml (hunk 11 above)
Comment (moarychan) [reply pom.xml]: No, I have updated the doc for Spring native parts in *spring-native/storage-blob-native/README.md*; the extra 6.x profiles are used to simulate the profile `native` and `nativeTest` in `spring-boot-starter-parent`, currently, we will not recommend users to use these features.

File: spring-native/storage-blob-native/README.md (hunk 21 above)
Comment (moarychan) [reply spring-native/storage-blob-native/README.md:181]: We will not recommend users use these features on Spring Cloud Azure 6.0, the extra xxx-6.x profiles are used to simulate the profile `native` and `nativeTest` in spring-boot-starter-parent:3.0.0.

File: spring-native/storage-blob-native/README.md (hunk 20 above)
Comment (moarychan) [reply spring-native/storage-blob-native/README.md:157]: The file `spring-native/README.md` is used to make the root directory display only one level of directory structure, such as `spring-native`, not `spring-native/storage-blob-native`.

File: pom.xml (hunk 19 above)
Comment (moarychan) [reply pom.xml:453]: > 1. Why nativeTest is only necessary for `6.x`?
   they are used to simulate the profile `native` and `nativeTest` in spring-boot-starter-parent:3.0.0.
> 2. Should we active this profile in GitHub actions?
   Not necessary.
> 3. Could you please add more description about spring-native related profiles? These profiles make the pom file too long, may make customer confused.
  The user should check each sample's README.md,  we have provided the steps to run the sample and related reference doc.

File: pom.xml (hunk 18 above)
Comment (moarychan) [reply pom.xml:406]: It's a copy form `spring-boot-starter-parent:3.0.0`, it just simulates the profile when using profile `spring-cloud-azure-6.x`.

image

File: pom.xml (hunk 15 above)
Comment (moarychan) [reply pom.xml:179]: It's used for `org.springframework.experimental:spring-native`, keeping the configuration the same with the project generated by start.spring.io

image

File: pom.xml (hunk 16 above)
Comment (moarychan) [reply pom.xml:251]: It may be too long, it can not use standalone, it should combine the `spring-cloud-azure-4.x` and `buildpack-4.x`.

File: spring-petclinic-microservices/pom.xml (hunk 22 above)
Comment (fangjian0423) [reply spring-petclinic-microservices/pom.xml:42]: It's used by [spring-boot-maven-plugin](https://github.com/Azure-Samples/azure-spring-boot-samples/blob/spring-cloud-azure_4.5.0/spring-petclinic-microservices/pom.xml#L128).

Without it, will compile error: 

image

------

Why we didn't define it before?

Because our project inherit from `spring-boot-starter-parent` at first. These 2 properties defined there, check it in [spring-boot-starter-parent pom](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-starter-parent/2.7.7/spring-boot-starter-parent-2.7.7.pom).

File: spring-petclinic-microservices/pom.xml (hunk 23 above)
Comment (fangjian0423) [reply spring-petclinic-microservices/pom.xml]: delete it.

Just make sure the version defined in root `pom.xml`:

```xml
<build>
    <plugins>
      <plugin>
        <groupId>org.springframework.boot</groupId>
        <artifactId>spring-boot-maven-plugin</artifactId>
        <version>${version.spring.boot}</version>
      </plugin>
      ...
    </plugins>
</build>
```

======

Why didn't it cause failure before this PR?

Because our project inherit from `spring-boot-starter-parent` at first, but now our project use scope import.

With maven you can only inherit pluginManagement when using that POM as a parent.
Scope import only brings you dependencyManagement. Check it [here](https://stackoverflow.com/a/40607508/4239209).

So we should use version to define `spring-boot-maven-plugin` version.

File: spring-native/storage-blob-native/README.md (hunk 21 above)
Comment (fangjian0423) [reply spring-native/storage-blob-native/README.md:181]: could we add the recommends to this docs?

File: README.md (hunk 13 above)
Comment (fangjian0423) [reply README.md:109]: add `Introduction to maven profiles` section to describe profile

File: spring-native/storage-blob-native/README.md (hunk 21 above)
Comment (moarychan) [reply spring-native/storage-blob-native/README.md:181]: This sample has been marked as not supporting Spring Cloud Azure 6.0, we will not say the native support with Spring Cloud Azure 6.0, but some samples actually can be built to executable with Spring Cloud Azure 6.0.

File: pom.xml (hunk 19 above)
Comment (chenrujun) [reply pom.xml:450]: How about writing this information into root README.md? For a customer, it's high possibility that he won't care about the content in `spring-native/xx/README.md`. He will be confused when he sees these contents in the root pom.xml. Put this information in the root README is one solution. Another solution is adding this information as comment in root pom.xml.

Comment (moarychan) [reply pom.xml:450]: Updated in root README.md file.

**Review by chenrujun — APPROVED** [review]
